sdk/appcenter-analytics/src/test/java/com/microsoft/appcenter/analytics/AnalyticsTest.java

```diff
@@ -367,6 +367,45 @@ public void trackEventWithInvalidFlags() {
         AppCenterLog.warn(eq(AppCenter.LOG_TAG), anyString());
     }
 
+    @Test
+    public void trackEventWithUserIdWhenConfiguredForTarget() {
+        when(mAppCenter.getUserId()).thenReturn("alice");
```

Comment:
nit:Can you use prefix for the userId when mocking the getter? Just to make things clear, nothing more than that.

Reply:
ok